ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Claims.scala

```diff
@@ -41,6 +41,12 @@ case object ClaimActAsAnyParty extends Claim
   */
 final case class ClaimActAsParty(name: Ref.Party) extends Claim
 
+/** Authorized to read all data for the given party.
+  *
+  * Does NOT authorize to modify the ledger in any way.
```

Comment:
```suggestion
  * Does NOT authorize to issue commands.
```
No one can ever _modify_ the ledger. :wink: :nerd_face:

Reply:
Well you can append to a ledger, which is a kind of modification? But your formulation is probably better 😃 .